dll/win32/syssetup/wizard.c

```diff
@@ -2041,6 +2041,12 @@ RegistrationProc(LPVOID Parameter)
     RegistrationNotify.Progress = RegistrationData->DllCount;
     RegistrationNotify.ActivityID = IDS_REGISTERING_COMPONENTS;
     RegistrationNotify.CurrentItem = NULL;
+
+    RegisterTypeLibraries(hSysSetupInf, L"TypeLibraries");
+
+    // FIXME: Move this call to a separate cleanup page!
+    RtlCreateBootStatusDataFile();
```

Comment:
Looks like this function call could stay where it originally was 🤔

Reply:
it is also something that should probably prevent the process from ending.

Reply:
in what sense? it just creates a bootstat.dat in C:\ReactOS; this should not prevent anything else? 🤔